### ISSUE TITLE
Release for SDK 2.0.0

### ISSUE THREAD

**Issue by Manu3l0us** [raspberrypi/pico-setup-windows#53]
I just took a Pico 2 out of the box and wanted to get things going. Unfortunately, there is not Windows installer release for SDK 2.0.0. Do you plan to provide that in the future?

**Comment by Manu3l0us** [comment]
Nevermind, just saw the note in the readme

**Comment by eriklundh** [comment]
I have used this installer to set up Pico-SDK for Pico development up until Pico-SDK 1.51.
I have now switched to https://github.com/raspberrypi/pico-vscode
One advantage of the pico-vscode plugin, and it's associated command line python utilities, is that it sets up your projects so that you don't have to start up with a special command file to get compilation to work. 
But I recommend that you learn of VSCode profiles, if you want to use the same VsCode install for any other purpose than Pico development, due to well-known issues with conflicting VsCode plugins.
